tests/view/crudapi/test_crud_api.py

```diff
@@ -30,3 +30,45 @@ def test_get_not_found_object(app):
         apiview = FakeAPIView()
         with pytest.raises(NotFound, message="Fake object not found"):
             apiview.get(0)
+
+
+def test_post_object(app):
+    """ Tests for CrudeAPIView post method. """
+    with app.test_request_context(
+            "/test/crudapitest"
+    ):
+        apiview = FakeAPIView()
+        payload = {"some_id": 6, "some_attr": "attr6"}
+        result = apiview.post(payload)
+    json_result = json.loads(result[0].get_data(as_text=True))
+    assert result[0].is_json is True
+    assert json_result == payload, "Wrong result returned from CrudeAPI view"
+    assert result[1] == HTTPStatus.OK, "Wrong response from CrudeAPI view"
+
+
+def test_put_object(app):
+    """ Tests for CrudeAPIView post method. """
+    with app.test_request_context(
+            "/test/crudapitest"
+    ):
+        apiview = FakeAPIView()
+        payload = {"some_id": 6, "some_attr": "attr6"}
+        result = apiview.put(payload)
+    json_result = json.loads(result[0].get_data(as_text=True))
+    assert result[0].is_json is True
+    assert json_result == payload, "Wrong result returned from CrudeAPI view"
+    assert result[1] == HTTPStatus.OK, "Wrong response from CrudeAPI view"
+
+
+def test_delete_object(app):
+    """ Tests for CrudeAPIView post method. """
```

Comment:
@soin08 can you add the following sentence to these dostrings: "FakeAPIView extends CrudeAPIView"? This way it will be more clear... right now it looks as if these tests are only testing a mock, not the actual CrudeAPIView

Reply:
Done